test/java/org/opendatakit/briefcase/util/BriefcaseVersionManagerTest.java

```diff
@@ -0,0 +1,60 @@
+package org.opendatakit.briefcase.util;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.opendatakit.briefcase.reused.http.RequestBuilder.get;
+import static org.opendatakit.briefcase.reused.http.RequestBuilder.url;
+import static org.opendatakit.briefcase.util.BriefcaseVersionManager.getLatestUrl;
+
+import org.junit.Test;
+import org.opendatakit.briefcase.reused.http.FakeHttp;
+import org.opendatakit.briefcase.reused.http.response.ResponseHelpers;
+
+public class BriefcaseVersionManagerTest {
+  @Test
+  public void knows_the_url_to_get_the_latest_version() {
+    assertThat(getLatestUrl(), is(url("https://github.com/opendatakit/briefcase/releases/latest")));
```

Comment:
Maybe this could be a constant in `BriefcaseVersionManager`?

Reply:
I'm going to leave this duplication. It's not a big deal, since it's just a getter but:
- Using a constant in the implementation would feel like cheating and make the test quite irrelevant (basically, testing a getter).
- This test just makes the url we're using explicit and serves as documentation of the tested class. One day, when GH changes the url, we can come back and know what has to be changes without investing a lot of time on it.